File: FusionAndroid/app/src/main/java/fr/intechinfo/fusionandroid/SMSList.java
package fr.intechinfo.fusionandroid;

import java.util.List;

public class SMSList {
    List<SMS> sms;

    public List<SMS> GetSMS(){
        return this.sms;
    }

    public void SetSMS(List<SMS> sms) {
        this.sms = sms;
    }
}

File: FusionAndroid/app/src/main/java/fr/intechinfo/fusionandroid/SMSList.kt
package fr.intechinfo.fusionandroid

class SMSList {
    internal var sms: List<SMS>? = null

    fun GetSMS(): List<SMS>? {
        return this.sms
    }

    fun SetSMS(sms: List<SMS>) {
        this.sms = sms
    }
}
